content/en/flux/installation/_index.md

```diff
@@ -0,0 +1,308 @@
+* [Azure DevOps](./bootstrap/azure.md#flux-installation-for-azure-devops)
+* [Bitbucket Server and Data Center](./bootstrap/bitbucket.md#bitbucket-server-and-data-center)
+* [GitHub.com and GitHub Enterprise](./bootstrap/github.md#github-and-github-enterprise)
+* [GitLab.com and GitLab Enterprise](./bootstrap/gitlab.md#gitlab-and-gitlab-enterprise)
+
+## Bootstrap with Terraform
+
+The bootstrap procedure can be implemented with Terraform using the Flux provider published on
+[registry.terraform.io](https://registry.terraform.io/providers/fluxcd/flux).
+The provider offers a Terraform resource called
+[flux_bootstrap_git](https://registry.terraform.io/providers/fluxcd/flux/latest/docs/resources/bootstrap_git)
+that can be used to bootstrap Flux in the same way the Flux CLI does it.
+
+Example of Git HTTPS bootstrap:
+
+```hcl
+provider "flux" {
+  kubernetes = {
+    config_path = "~/.kube/config"
+  }
+  git = {
+    url  = var.gitlab_url
+    http = {
+      username = var.gitlab_user
+      password = var.gitlab_token
+    }
+  }
+}
+
+resource "flux_bootstrap_git" "this" {
+  path                   = "clusters/my-cluster"
+  network_policy         = true
+  kustomization_override = file("${path.module}/kustomization.yaml")
+}
+```
+
+For more details on how to use the Terraform provider
+please see the [Flux docs on registry.terraform.io](https://registry.terraform.io/providers/fluxcd/flux/latest/docs).
+
+## Customize Flux manifests
+
+You can customize the Flux components before or after running bootstrap.
+
+Assuming you want to customise the Flux controllers before they get deployed on the cluster,
+first you'll need to create a Git repository and clone it locally.
+
+Create the file structure required by bootstrap with:
+
+```sh
+mkdir -p clusters/my-cluster/flux-system
+touch clusters/my-cluster/flux-system/gotk-components.yaml \
+    clusters/my-cluster/flux-system/gotk-sync.yaml \
+    clusters/my-cluster/flux-system/kustomization.yaml
+```
+
+Add patches to `kustomization.yaml`:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1
+kind: Kustomization
+resources: # manifests generated during bootstrap
+  - gotk-components.yaml
+  - gotk-sync.yaml
+
+patches: # customize the manifests during bootstrap
+  - target:
+      kind: Deployment
+      labelSelector: app.kubernetes.io/part-of=flux
+    patch: |
+      # strategic merge or JSON patch
+```
+
+Push the changes to main branch:
+
+```sh
+git add -A && git commit -m "init flux" && git push
+```
+
+And run the bootstrap for `clusters/my-cluster`:
+
+```sh
+flux bootstrap git \
+  --url=ssh://git@<host>/<org>/<repository> \
+  --branch=main \
+  --path=clusters/my-cluster
+```
+
+To make further amendments, pull the changes locally,
+edit the `kustomization.yaml` file, push the changes upstream
+and rerun bootstrap or let Flux upgrade itself.
+
+Checkout the [bootstrap cheatsheet](../cheatsheets/bootstrap) for various examples of how to customize Flux.
+
+### Multi-tenancy lockdown
```

Comment:
This should be in the "Configuration/Multi-tenancy lockdown" page.

Reply:
Noted. You will see it on the configuration page when I create a PR for it.